clinic/clinic-user-guide-for-tiup.md

```diff
@@ -0,0 +1,345 @@
+
+#### Method 2. Pack and upload data
+
+If the network where your cluster is located cannot directly connect to the Clinic Server, you need to pack the data on your intranet and upload it to the Clinic Server using a network-connected device. The detailed operations are as follows:
+
+1. Compress and encrypt the data package collected in [Step 2. Collect data](#step-2-collect-data).
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    tiup diag package ${filepath}
+    ```
+
+    When packaging, Diag encrypts and compresses the data at the same time. In the test environment, 800 MB of data was compressed to 57 MB. The following is an example output:
+
+    ```bash
+    Starting component `diag`: /root/.tiup/components/diag/v0.5.1/diag package diag-fNTnz5MGhr6
+    packaged data set saved to /home/qiaodan/diag-fNTnz5MGhr6.diag
+    ```
+
+    After the data is packaged through the `package` command above, the data is in the `.diag` format. This file can only be decrypted and viewed after uploading it to the Clinic Server. If you need to directly forward the collected data instead of viewing it in the Clinic Server, you can compress the data by your own method and forward it.
+
+2. From a machine with the network connection to the Clinic Server, upload the compressed data package.
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    tiup diag upload ${filepath} -u=username -p='password'
+    ```
+
+    > **Note:**
+    >
+    > TiDB Clinic is currently in the Beta testing stage for invited users only. If you need to upload data to the Clinic Server using Diag, contact [PingCAP technical support](https://en.pingcap.com/contact-us/) to get a trial account first.
+
+    The following is an example output:
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    [root@Copy-of-VM-EE-CentOS76-v1 qiaodan]# tiup diag upload /home/qiaodan/diag-fNTnz5MGhr6
+    Starting component `diag`: /root/.tiup/components/diag/v0.5.1/diag upload /home/qiaodan/diag-fNTnz5MGhr6
+    Enter Username: username
+    Enter Password: >>>>>>>>>>>>>>>>>>>>>>>>>>>>>><>>>>>>>>>
+    Completed!
+    Download URL: "https://clinic.pingcap.com:4433/diag/files?uuid=XXXX"
+    ```
+
+    After the upload is complete, you need to send the data access link of `Download URL` to the PingCAP technical support staff you contacted before.
+
+    > **Note:**
+    >
+    > Currently, **only** PingCAP technical support staff can open the data access link. External users cannot open the link.
+
+## Perform a quick check on the cluster status locally
+
+You can have a quick check on the cluster status locally using Diag. Even if your cluster runs stably now, it is necessary to periodically check the cluster for potential stability risks. The TiDB Clinic Beta version provides a rationality check on cluster configuration items to discover unreasonable configurations and provide modification suggestions.
```

Comment:
```suggestion
You can have a quick check on the cluster status locally using Diag. Even if your cluster runs stably now, it is necessary to periodically check the cluster to avoid potential stability risks. The TiDB Clinic Beta version provides a rationality check on cluster configuration items to discover unreasonable configurations and provide modification suggestions.
```